client/blocks/post-share/index.jsx

```diff
@@ -423,10 +440,20 @@ class PostShare extends Component {
 			);
 		}
 
-		if ( this.props.schedulingFailed ) {
+		if ( schedulingFailed ) {
 			return (
-				<Notice status="is-error" onDismissClick={ this.dismiss }>
-					{ translate( "Scheduling share failed. Please don't be mad." ) }
+				<Notice
+					status="is-error"
+					onDismissClick={ this.dismiss }
+					text={ translate( 'Scheduling share failed.' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.